rules/phpdocs_basic.php

```diff
@@ -176,51 +174,6 @@ function local_moodlecheck_phpdoccontentsinlinetag(local_moodlecheck_file $file)
     return $errors;
 }
 
-/**
- * Makes sure that file-level phpdocs and all classes have one-line short description
- *
- * @param local_moodlecheck_file $file
- * @return array of found errors
- */
-function local_moodlecheck_phpdocsfistline(local_moodlecheck_file $file) {
-    $errors = [];
-
-    if (($phpdocs = $file->find_file_phpdocs()) && !$file->find_file_phpdocs()->get_shortdescription()) {
```

Comment:
Maybe we can already remove the `-> get_shortdescription()` method completely? Unless I'm wrong, there aren't more uses. Not critical, just saw it.

Reply:
Thanks - I actually saw that just before you suggested it.

Fixed.